node/pkg/tss/crypto.go

```diff
@@ -25,12 +25,26 @@ func hashSignedMessage(msg *tsscommv1.SignedMessage) digest {
 		return digest{}
 	}
 
-	b := bytes.NewBuffer(nil)
-	b.Write(msg.Content.Payload)
-	vaa.MustWrite(b, binary.BigEndian, msg.Content.MsgSerialNumber)
-	pid := msg.Sender
-	b.Write([]byte(pid.Id))
-	b.Write(pid.Key)
+	var b *bytes.Buffer
+
+	switch m := msg.Content.(type) {
```

Comment:
I can't. The SignedMessage type is a protobuf message, and Wormhole doesn't like putting them in the same package as the code that uses the generated messages.
As a result, I can't edit the SignedMessage functionality.
In addition, I can't rely on Protobuf's marshal method since it doesn't guarantee it'll be canonic marshalling.